orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/MonitorKatoTask.groovy

```diff
@@ -149,7 +149,11 @@ class MonitorKatoTask implements RetryableTask {
       }
       katoTasks << m
       outputs["kato.tasks"] = katoTasks
+    }
 
+    if (status == ExecutionStatus.TERMINAL && katoTask.status.retryable) {
+      kato.resumeTask(katoTask.id)
+      status = ExecutionStatus.RUNNING
```

Comment:
probably worth at least logging something here that we are "going to retry".
You are basically hiding an error from the user and it is possible the error goes away on retry or it becomes another error... and that makes it harder to debug issues. (this is the thing we talked about some standardization in orca about error messages... someday)

Reply:
You're right. I've updated the PR to show usage of the `SystemNotification` stuff I was talking about.